VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.ts

```diff
@@ -135,4 +136,23 @@ export class ParticipantStatusComponent implements OnInit {
                 return 'participant-default-status';
         }
     }
+
+    private sortParticipants() {
+        const judges = this.participants.filter(participant => participant.hearingRole === 'Judge');
+        const panelMembersAndWingers = this.participants.filter(participant =>
+            ['Panel Member', 'Winger'].includes(participant.hearingRole)
```

Comment:
Use the HearingRole class variables rather than hard-coding Panel Member and winger

Reply:
updated to use HearingRole